src/app/(home)/globals.css

```
@@ -1,3 +1,4 @@
+@import url("//spoqa.github.io/spoqa-han-sans/css/SpoqaHanSansNeo.css");
```

Stylelint (17.4.0)

[error] Expected "url("//spoqa.github.io/spoqa-han-sans/css/SpoqaHanSansNeo.css")" to be ""//spoqa.github.io/spoqa-han-sans/css/SpoqaHanSansNeo.css"" (import-notation)

(import-notation)